pkg/kopia/command/storage/common_utils.go

```diff
@@ -0,0 +1,16 @@
+package storage
+
+import (
+	"path"
+)
+
+// GenerateFullRepoPath defines the manner in which a location-specific prefix
```

Comment:
Is this only used in the commands or outside of commands? If used outside, I would move the file to `kopia/common_utils.go`

Reply:
In that case, we would have to create a separate child(utils) package otherwise parent to child imports could later result in cyclic import issues.